scripts/deploy.sh

```diff
@@ -39,6 +41,7 @@ echo "Deploying new subgraph for court address ${COURT} to network ${NETWORK}"
 cp subgraph.template.yaml subgraph.yaml
 sed -i -e "s/{{network}}/${ENV}/g" subgraph.yaml
 sed -i -e "s/{{court}}/${COURT}/g" subgraph.yaml
+rm subgraph.yaml-e
```

Comment:
What's this for? It fails for me.

Reply:
I think `sed` works differently in macOS then, I added a `-f` so it doesn't fail for you

Reply:
`sed`? Is that trailing `-e` a `sed` command? I have never seen this before. What are you trying to delete here? Is this POSIX or any other standard?

Reply:
No, I mean that `sed -e` is generating an output to me and maybe it is not doing the same for you, making the `rm` step fails for you.

Reply:
So those previous `sed` commands are generating a file called `subgraph.yaml-e`? That's weird. Maybe we should fix them instead. Actually, what do you need the `-e` flag there (in `sed` commands I mean)? It works without it for me.

Reply:
Yes, I think that's the problem. My `man sed` says:
```
-i[SUFFIX], --in-place[=SUFFIX]
```
I guess for you it works with a space too, so it's considering `-e` as the suffix for the new file. Try removing `-e` in `sed` command.

Reply:
The thing is that `sed -i` in macOS is not POSIX but FreeBSD, and the difference is that it expects an output path. Removing the `-e` will not work in macOS since you will have to specify something and will stop working for non-mac users. See [here](https://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed#answer-131940):

> `-i ''` works with OS X's sed but not GNU `sed`. `-i` works with GNU sed, recent versions of NetBSD, OpenBSD sed, but not OS X's `sed`. `-i -e` works with both but in the case of FreeBSD `sed` makes a backup of the original file with `-e` appended to the file name (and you need to pass no more than one expression to `sed`).

Reply:
What a dirty hack! :joy:

Reply:
Man I was having this exact issue with `sed` over the weekend and wondering what the hell was going on with `-i` on macOS.

Reply:
Perhaps this is something we leave as a comment though :).

Reply:
Yes,  remember having this problem with Jorge long time ago, but I didn't remember the workaround.